sdk/template/azure_template/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "azure_template"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
Sure you want to "GA" this just yet?

Reply:
I can pick another minor.  We just need to keep it at something that's never been released and 0.1.0 is burned now

Reply:
I might make more updates with examples that are not covered in azure_core, I have a draft PR too.
Unsure if that information affects this.